profiles/models.py

```diff
@@ -67,7 +68,7 @@ def get_notification_context(self):
 
 @reversion.register()
 class Profile(UUIDModel):
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    user = models.OneToOneField(User, on_delete=models.CASCADE, null=True, blank=True)
```

Comment:
would it make sense to `PROTECT` here? I think `Profile` is more important than `User`  🤔

Reply:
but I am open to `CASCADE` as well...

Reply:
Maybe `PROTECT` is better for now. This `User`-`Profile` relationship might go through some heavy architecture changes anyways depending on tunnistamo-related stuff.